app/controllers/hub/metrics_controller.rb

```diff
@@ -0,0 +1,25 @@
+module Hub
+  class MetricsController < ApplicationController
+    include AccessControllable
+    before_action :require_sign_in
+    layout "admin"
+    load_and_authorize_resource :vita_partner, parent: false
+
+    def index
+      # Run SLA queries and cache the result for 10 minutes.
+      # The view will only display values for vita partners the user has access to view.
+      data = Rails.cache.fetch("metrics/sla_breaches/attention_needed", expires_in: 10.minutes) do
```

Comment:
10 minute server cache on the SLA breach data -- enough to ensure we're not re-running this query on every page load or for every User who accesses the metrics, but often enough to make sure the data is useful to those who access it.

Reply:
Neat